game/game.go

```diff
@@ -372,3 +373,13 @@ func (g *Game) attackWithEveryone() {
 		g.TakeAction(actions[0])
 	}
 }
+
+// playManaAbilityAction plays the first mana ability action
+func (g *Game) playManaAbilityAction() {
+	for _, a := range g.Priority().ManaActions() {
+		g.TakeAction(a)
+		return
+	}
+	g.Print()
+	panic("playInstant failed")
```

Comment:
an odd failure message since it is in `playManaAbilityAction` not in `playInstant`